VideoLocker/src/androidTest/java/org/edx/mobile/test/http/MockApiTests.java

```diff
@@ -0,0 +1,209 @@
\": {\"facebook\": null}, \"subscription_id\": \"course_IRSWYZTULAXUKVBOGMYDGNCUKUXTGVBSGAYTI___\", \"course_image\": \"/c4x/DelftX/ET.3034TU/asset/images_course_image.jpg\"}, \"certificate\": {}}, {\"created\": \"2014-09-19T19:18:30Z\", \"mode\": \"honor\", \"is_active\": true, \"course\": {\"course_about\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX.1/2014/about\", \"course_updates\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX.1/2014/updates\", \"number\": \"DemoX.1\", \"org\": \"edX\", \"video_outline\": \"" + baseMockUrl + "/api/mobile/v0.5/video_outlines/courses/edX/DemoX.1/2014\", \"id\": \"edX/DemoX.1/2014\", \"latest_updates\": {\"video\": null}, \"end\": null, \"name\": \"Demo Course\", \"course_handouts\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX.1/2014/handouts\", \"start\": \"2014-08-01T04:00:00Z\", \"social_urls\": {\"facebook\": null}, \"subscription_id\": \"course_MVSFQL2EMVWW6WBOGEXTEMBRGQ______\", \"course_image\": \"/c4x/edX/DemoX.1/asset/CourseListingImage.png\"}, \"certificate\": {}}, {\"created\": \"2014-09-19T19:16:13Z\", \"mode\": \"honor\", \"is_active\": true, \"course\": {\"course_about\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/WellesleyX/SOC108x/2014_SOND/about\", \"course_updates\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/WellesleyX/SOC108x/2014_SOND/updates\", \"number\": \"SOC108x\", \"org\": \"WellesleyX\", \"video_outline\": \"" + baseMockUrl + "/api/mobile/v0.5/video_outlines/courses/WellesleyX/SOC108x/2014_SOND\", \"id\": \"WellesleyX/SOC108x/2014_SOND\", \"latest_updates\": {\"video\": null}, \"end\": \"2014-11-10T23:30:00Z\", \"name\": \"Introduction to Global Sociology\", \"course_handouts\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/WellesleyX/SOC108x/2014_SOND/handouts\", \"start\": \"2014-09-02T19:00:00Z\", \"social_urls\": {\"facebook\": null}, \"subscription_id\": \"course_K5SWY3DFONWGK6KYF5JU6QZRGA4HQLZSGAYTIX2TJ5HEI___\", \"course_image\": \"/c4x/WellesleyX/SOC108x/asset/images_course_image.jpg\"}, \"certificate\": {}}, {\"created\": \"2014-08-25T16:38:08Z\", \"mode\": \"honor\", \"is_active\": true, \"course\": {\"course_about\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX/Demo_Course/about\", \"course_updates\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX/Demo_Course/updates\", \"number\": \"DemoX\", \"org\": \"edX\", \"video_outline\": \"" + baseMockUrl + "/api/mobile/v0.5/video_outlines/courses/edX/DemoX/Demo_Course\", \"id\": \"edX/DemoX/Demo_Course\", \"latest_updates\": {\"video\": null}, \"end\": null, \"name\": \"edX Demonstration Course\", \"course_handouts\": \"" + baseMockUrl + "/api/mobile/v0.5/course_info/edX/DemoX/Demo_Course/handouts\", \"start\": \"2013-02-05T05:00:00Z\", \"social_urls\": {\"facebook\": null}, \"subscription_id\": \"course_MVSFQL2EMVWW6WBPIRSW2327INXXK4TTMU______\", \"course_image\": \"/c4x/edX/DemoX/asset/images_course_image.jpg\"}, \"certificate\": {}}]");
```

Comment:
is that possible to use some type of template for generating json string object?
there are some template libraries,  maybe we can just use string template, or StringBuilder to  refactoring code?